source

```
@@ -85310,9 +85334,9 @@ interface <dfn interface>History</dfn> {
       entry</var>, then run these substeps:</p>
 
       <ol>
-       <li><p><span>Prompt to unload</span> the <span>active document</span> of the <var>specified
-       browsing context</var>. If the user <span>refused to allow the document to be
-       unloaded</span>, then return.</p></li>
+       <li><p>If the result of calling <span>prompt to unload</span> with the
+       <span>active document</span> of the <var>specified browsing context</var> is
+       "<code data-x="">refuse</code>", then return.</p></li>
```

Preexisting problem, but if you could fix while you're here it'd be nice: since we're in a queued task, we use "abort these steps" instead of "return"

---

Fixed

---

I think I commented on the wrong line. "close a browsing context" should be "return"; "traverse the history by a delta" should be "abort these steps".